Diagram and blog post

typaulhus opened the pull request:
**DO'D**
- added diagram to homepage 
- added blog post (stole the illustration Dan made for the post on appland.com. )
- hid `/docs` from the nav until this is ready/discarded

**TODO:** 
- add moar content to homepage (including links to Ruby & Java clients)
- update blog illustration if needed
- update GH link in nav (which repos are moving to new org?)

![Screen Shot 2020-10-14 at 5 34 19 PM](https://user-images.githubusercontent.com/123787/96047801-a512c300-0e43-11eb-8e86-639dcfcbe68b.png)
![Screen Shot 2020-10-14 at 5 34 26 PM](https://user-images.githubusercontent.com/123787/96047811-a7751d00-0e43-11eb-962f-9a1ed24d994d.png)
![Screen Shot 2020-10-14 at 5 34 32 PM](https://user-images.githubusercontent.com/123787/96047817-a8a64a00-0e43-11eb-879f-36beda3726c0.png)

kgilpin commented:
Where does the GitHub link at the top go? How about GitHub.com/applandinc

kgilpin commented on _includes/navigation.html:
```diff
@@ -1,7 +1,8 @@
-    <li><a href="/docs">Docs</a></li>
+    <!-- <li><a href="/docs">Docs</a></li> -->
     <li><a href="/blog">Blog</a></li>
+    <li><a class="gh-link" href="https://github.com/applandinc/appmap">Github</a></li>
```
Link to https://github.com/applandinc

typaulhus commented:
> Where does the GitHub link at the top go? How about GitHub.com/applandinc

I updated the GH link. Also updated the text in the top section to match what Dan had used.  (AppLand.org instead of AppMap)
